src/AsyncEventSource.h

```diff
@@ -21,22 +21,29 @@
 #define ASYNCEVENTSOURCE_H_
 
 #include <Arduino.h>
+#include <string>
 #ifdef ESP32
   #include <AsyncTCP.h>
   #include <mutex>
   #ifndef SSE_MAX_QUEUED_MESSAGES
     #define SSE_MAX_QUEUED_MESSAGES 32
   #endif
+  #define SSE_MIN_INFLIGH 2*1460      // allow 2 MSS packets
+  #define SSE_MAX_INFLIGH 16*1024     // but no more than 16k, no need to blow it, since same data is kept in local Q
```

Comment:
- Could you please write some comments to explain to the users what this new settings does and what effect they have ?
- why 16 
- Did you perform some concurrent tests with different factors ?
- Also, should it be customizable by the user ? it cannot be here like `SSE_MAX_QUEUED_MESSAGES` ?
- If customizable, also add a check macro to ensure the correctness;

#if SSE_MAX_INFLIGH <= SSE_MIN_INFLIGH
  #error "..."
#endif

Reply:
pls see my concerns above, still unclear what to do with that.